### PR TITLE
private locations: take another pass

### PR DESCRIPTION
**Pull request by benben**
## Affected Components
* [x] Docs

## Notes for the Reviewer
Improved Private Locations and Checkly Agent docs to make them more concise and precise.

### CODE REVIEW

**Review by tnolet — CHANGES_REQUESTED** [review]
Two nits:
- extra space after period
- can you remove the stray ` backticks on all the code blocks? There are a couple

File: site/content/docs/private-locations/_index.md [hunk 1]
```diff
@@ -11,27 +11,21 @@ aliases:
 cli: true
 ---
 
-A Private Location is similar to our existing public locations, but you have control over it. A Private Location can be
-in your own segregated network, on-premises or cloud-based. You need one or more Checkly Agents installed in that location/network
-with access to your applications.
+A Private Location is an additional location in Checkly. It acts exactly like any other location, but you run it on your own infrastructure.A Private Location can be used to run checks on your services which are not direcrtly exposed to the public internet but in your own segregated network. To operate a Private Location you need to run at least one Checkly Agent.
```
Comment (benben) [inline site/content/docs/private-locations/_index.md:14]:
```suggestion
A Private Location is an additional location in Checkly. It acts exactly like any other location, but you run it on your own infrastructure. A Private Location can be used to run checks on your services which are not direcrtly exposed to the public internet but in your own segregated network. To operate a Private Location you need to run at least one Checkly Agent.
```

File: site/content/docs/private-locations/checkly-agent-configuration.md [hunk 2]
```diff
@@ -93,7 +91,7 @@ API keys can be rotated as necessary in order to maintain good security practice
 
    ![two keys](/docs/images/private-locations/two_keys.png)
 
-6. You now need to replace your Checkly agents. You can do this one-by-one or as a group, just make sure you always have 
+6. You now need to replace your Checkly agents. You can do this one-by-one or as a group, just make sure you always have
 enough agent containers up and running based on your workload. Using your container management tool, start new agents with the new API key:
 
 ```bash
```
Comment (tnolet) [inline site/content/docs/private-locations/checkly-agent-configuration.md:97]: Still some trailing ` in this one

**Review by tnolet — CHANGES_REQUESTED** [review]
one more nit

File: site/content/docs/private-locations/checkly-agent-configuration.md [hunk 3]
```diff
@@ -93,7 +91,7 @@ API keys can be rotated as necessary in order to maintain good security practice
 
    ![two keys](/docs/images/private-locations/two_keys.png)
 
-6. You now need to replace your Checkly agents. You can do this one-by-one or as a group, just make sure you always have 
+6. You now need to replace your Checkly agents. You can do this one-by-one or as a group, just make sure you always have
 enough agent containers up and running based on your workload. Using your container management tool, start new agents with the new API key:
```
Comment (benben) [inline site/content/docs/private-locations/checkly-agent-configuration.md:95]:
```suggestion
6. You now need to replace your Checkly agents. You can do this one-by-one or as a group, just make sure you always have enough agent containers up and running based on your workload. Using your container management tool, start new agents with the new API key:
```

**Review by tnolet — APPROVED** [review]
LGTM!

**Pull request merged**